README.md

```diff
@@ -4,54 +4,88 @@ A CLI to co-host websites published to [IPFS].
 
 If your domain has a [DNSlink] to a CID, then `ipfs-cohost` will let others pin it to their IPFS node.
 
-
 ## Usage
 
-Pass it the list of domains you want to cohost. It finds the CID from the [DNSLink] for that domain, and pins it to your local IPFS node.
+Pass it the list of domains you want to cohost. It finds the CID from the [DNSLink] for that domain and starts cohosting the domains. By default, we use a [lazy cohosting]. You can force the domains to be fully cohosted by using the flag `--full`.
 
 ```console
-$ ipfs-cohost ipfs.io docs.ipfs.io awesome.ipfs.io
+$ ipfs-cohost add ipfs.io docs.ipfs.io awesome.ipfs.io
 🔌 Using local ipfs daemon via http api
-🔍 Finding DNSLinks for 3 domains
-🔗 ipfs.io         QmXZz6vQTMiu6UyGxVgpLB6xJdHvvUbhdWagJQNnxXAjpn 11.5 MB
-🔗 docs.ipfs.io    QmXrsvjeZeH6rCzgQSJycKq9fFqNgkptTqYRexzaNy4wx3 6.59 MB
-🔗 awesome.ipfs.io QmdgXaAryZpe3vQcHyhzk5kowbgxvY3XaGTYvYAhoBrHLm 7.45 MB
-📦 Total size 25.6 MB for 3 domains
-📍 Pinned ipfs.io
-📍 Pinned docs.ipfs.io
-📍 Pinned awesome.ipfs.io
+📍 docs.ipfs.io    QmNrbogjGZWgUSrbmHXydwc5b51oJQsBfHA2RkNRVf2ikc 6.86 MB
+📍 awesome.ipfs.io QmPHrA6RT2j7bEcBrdeV2z6ZWrchDguw9wBchHPr6VZcFS 7.6 MB
+📍 ipfs.io         QmYb3dbymigAPcaiQUgnPxbwgMKjvxVBrrxSvFHwbZLVkq 10 MB
+📦 Total size 24.5 MB for 3 domains
 🤝 Co-hosting 3 domains via IPFS.
 ```
 
-Passing the `--no-pin` flag you can find the total size but skip pinning it.
+Passing the `--silent` options will prevent any logging.
 
 ```console
-$ ipfs-cohost dist.ipfs.io tr.wikipedia-on-ipfs.org --no-pin
+$ ipfs-cohost ipfs.io --silent
```

Comment:
Should be `$ ipfs-cohost add ipfs.io --silent`?

Reply:
Can be both ways, but yes, will update!